src/regolith/exemplars.json

```diff
@@ -2169,6 +2169,49 @@
             "year": 2020
         }
     ],
+    "releaseReports": [
+        {
+            "_id": "sbillinge",
+            "date": "17/07/2025",
+            "releases": [
+                {
+                "package_name": "regolith",
+                "description": "A Group Content Management System",
+                "url": "https://github.com/regro/regolith",
+                "release_updates": [
+                    {
+                        "version_number": "0.9.1",
+                        "updated_capacities": ["Rearrange Regolith package to new Billingegroup package standards",
+                            "change name and then fix tests for function that formats awards and honors"],
+                        "release_url": "https://github.com/regro/regolith/releases/tag/0.9.1",
```

Comment:
maybe keep this if we can use it get the release info.  In which case, don't save the other information here, for example the changelog.

Maybe have a `release_type` that can be `major`, `minor`, `patch`.  This will allow us to chosse what levels of release we want to list in some situation.